### DISCUSSION TITLE
When will 4.3.4.15595 be available from the preservation project?

### DISCUSSION THREAD

**Discussion by Snify89** [TrinityCore/TrinityCore#29756] [General]
The repository 
https://github.com/The-Cataclysm-Preservation-Project/TrinityCore

has been recently archived with the following note:

> Archived repository for WoW 4.3.4.15595. 
The project will be reworked for Cataclysm Classic as an official new branch of TrinityCore at https://github.com/TrinityCore/TrinityCore

When can we except a (new) branch for 4.3.4.15595 in here?
And what are the plans for 4.3.4.15595 in general?

Thank you for your answers.

Edit: I just noticed the Cataclysm "Classic".
Does this mean that the previous "original" version won't be supported anymore?

**Comment by ghost** [comment]
https://wowclassic.blizzard.com/en-us/#upgrades

- ¹ Cataclysm Classic launches on or before August 31, 2024.

Nothing much will happen until September 2024, by the looks of it.
